freeglut/freeglut/src/fg_callbacks.c

```diff
@@ -126,6 +126,7 @@ IMPLEMENT_GLUT_CALLBACK_FUNC_ARG0_2NAME(MenuDestroy, Destroy)
 
 /* Implement all these callback setter functions... */
 IMPLEMENT_CURRENT_WINDOW_CALLBACK_FUNC_ARG2(Position)
+IMPLEMENT_CURRENT_WINDOW_CALLBACK_FUNC_ARG3_USER(KeyboardExt,int,int,int)
```

Comment:
Please add the public function for the `glutKeyboardExtFuncUcall` to `freeglut_ucall.h`